### ISSUE TITLE
Distributed Permanent Identifier Registry

### ISSUE THREAD

**Issue by harlantwood** [WebOfTrustInfo/rebooting-the-web-of-trust#78]
What is your level of interest/availability in working on the _Distributed Permanent Identifier Registry_ paper @jbenet @ChristopherA @aquabu @peacekeeper @talltree @dukedorje ?

I think we have the seed of something awesome here, but it needs a fair bit of work.

https://github.com/WebOfTrustInfo/rebooting-the-web-of-trust/blob/master/draft-documents/Distributed-Permanent-Identifier-Registry.md

**Comment by peacekeeper** [comment]
I agree it's a super fascinating idea (let's call it DPIR?) and we should try to "complete" it, even if it ends up only explaining the idea on a high level without many technical details.

My understanding from the session was that this is an abstract "meta" protocol (more social than technical) for registering and recovering an ID through multiple technologies. In contrast, the DPKI paper describes concrete technologies for registering IDs. So from a DPIR perspective, DPKI can be considered to provide one or more "sync protocols".

XDI could then perhaps be thought of as a concrete application (among others) that uses DPIR and DPKI.

Not sure what the next step is here, any suggestions? I'll start by re-reading both the DPIR and DPKI papers..

**Comment by talltree** [comment]
Harlan,

I'm traveling this week so have limited bandwidth until this weekend, but I
love this concept and I agree with Markus that it has great synergy with
the DPKI paper which I have been working on quite a bit with Greg and crew.

So I definitely would like to help with the DPIR paper as well. What's the
next milestone? Can you wait until this weekend to get feedback from me?

Thanks,

=Drummond

**Comment by jbenet** [comment]
Spoke with @ChristopherA today, plan:
- (1) write a paper describing (a) the constraints of the problem and (b) sketching the solution space. 
- (2) write a paper with a solution that works.

Need a real, solid solution before writing it up. I have time for writing (1) before Jan 1. And will be discussing (2) meanwhile, but doubt a paper for (2) will be out before Jan 1 (i want to make a thing that people like djb, dm, and dabo would vet.)

**Comment by harlantwood** [comment]
Awesome!  Happy to help, especially with brainstorming & editing.

**Comment by talltree** [comment]
Ditto from me. I want to help see this get implemented in early 2016.

**Comment by harlantwood** [comment]
@jbenet do you still see having the bandwidth to work on this?  What kind of help & support would you most like from the rest of us?

**Comment by talltree** [comment]
FWIW, Christopher and I are still very active on this topic—I want to see
a decentralized identifier registry implemented in 2016. I'd love to sync
up with Juan and others on it when they come up for air.

**Comment by dukedorje** [comment]
I am also keen to help out in whatever ways I can.
